.github/workflows/merge-pr.yml

```diff
@@ -62,3 +62,33 @@ jobs:
         working-directory: js/api-augment/dist
         env:
           NODE_AUTH_TOKEN: ${{secrets.NODE_AUTH_TOKEN}}
+
+  calc-code-coverage-main:
```

Comment:
This is to make sure that CodeCov gets the main branch information. Without this PRs get strange results as it is comparing against old stuff.